openapi-manager-example/apis/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "openapi-manager-example-apis"
+version = "0.1.0"
+edition = "2021"
+license.workspace = true
```

Comment:
Do we want to license the example under 0BSD so that people who copy parts of it (as we recommend) don't have to maintain attribution?

Reply:
Confirmed with bcantrill that 0BSD is fine for this example.